tests/src/test/fdc3.broadcast.ts

```diff
@@ -13,272 +13,554 @@ export default () =>
         instrument: true,
         contact: false,
       },
+      joinAppChannel: false,
+      broadcastMultipleItems: false,
     };
 
-    afterEach(async () => {
-      if (listener !== undefined) {
-        await listener.unsubscribe();
-        listener = undefined;
+    beforeEach(async () => {
+      await unsubscribeListeners();
+      if (channelsAppContext.joinAppChannel === false) {
+        await window.fdc3.leaveCurrentChannel();
```

Comment:
I don't see the harm in just running this unconditionally?

Reply:
it doesn't do anything when running app channel tests. It only works on user channels. But I guess I could just leave it anyway